shopyo/api/templates.py

```diff
@@ -0,0 +1,5 @@
+from flask import render_template
+
+
+def yo_render(template, context_dict):
+    return render_template(template, **context_dict)
```

Comment:
@Abdur-rahmaanJ doesn't `flask.render_template` already accept context ? source: https://flask.palletsprojects.com/en/1.1.x/api/#flask.render_template. So what is the point of doing this (wrapping) rather than directly calling `render_template` with context?

Reply:
@rehmanis is it context or **contect?

Reply:
**context so dictionary (key value).

Reply:
@rehmanis Exactly. If you do
```
return render_template('blogus/render_demo.html', context)
```
it gives out

```
    return render_template('blogus/render_demo.html', context)
TypeError: render_template() takes 1 positional argument but 2 were given
```

It is **context, not context

Reply:
Yes so why not do `template_render('blogus/render_demo.html', context=context)` ?  or `template_render('blogus/render_demo.html', **context)`

Reply:
@rehmanis I've been doing those since long, having this simple wrapper improves the dev experience.

Reply:
hmm not sure about this since it is adding **context vs context (plus tests etc) but I will trust your judgement.